resolvers/event.js

```diff
@@ -9,6 +9,53 @@ const eventResolvers = {
   },
   Event: {
     createdBy: (obj, args, context, info) => db.user.findByPk(obj.createdBy)
+  },
+  Mutation: {
+    addEvent: async (parent, args) => {
+      // Check if createdBy is valid
+      const user = await db.user.findByPk(args.createdBy);
+      if (!user) {
+        throw new Error("Invalid user ID");
+      }
+      return db.event.create({
+        name: args.name,
+        eventDate: args.eventDate,
+        createdBy: args.createdBy,
+        isActive: args.isActive
+      });
+    },
+    updateEvent: async (parent, args) => {
+      const event = await db.event.findByPk(args.id);
+      const user = args.createdBy
+        ? await db.user.findByPk(args.createdBy)
+        : true;
```

Comment:
is this ternary necessary? more specifically should `args.createdBy` ever be falsey?

Reply:
if somebody tries to update an event but doesn't pass in the `createdBy` field (i.e. they're not updating the createdBy field), then `args.createdBy` would be undefined (falsey). I'm not sure if this is the intended functionality - I just bypassed the problem by setting user to `true` when no createdBy argument is passed.

Reply:
gotcha, in that case something like this would be better - it's generally bad practice for a variable to possibly be two different types
```suggestion
      if (args.createdBy) {
        const user = await db.user.findByPk(args.createdBy);
        if (!user) {
          throw new Error("Invalid user ID");
        }
      }
```